test/runtime/Makefile

```diff
@@ -40,6 +41,12 @@ PROGRAMS= \
 	write \
 	writev
 
+SRCS-futex_test= \
+	$(CURDIR)/futex_test.c \
+	$(SRCDIR)/unix_process/ssp.c \
+LDFLAGS-futex_test=	-static
```

Comment:
The `\` character after ssp.c shouldn't be there, because it causes the LDFLAGS line to not be parsed correctly, which means that this program doesn't run in Nanos with the provided manifest file.
To run this program in Nanos, from the top-level folder of the repo execute `make run TARGET=futex_test`

Also, this should be in alphabetical order

Reply:
Hello @francescolavra, I removed the `\ `character on this line. However, when I run `make run TARGET=futex`, I get the following error: 
```
Could not access KVM kernel module: No such file or directory
qemu-system-x86_64: failed to initialize KVM: No such file or directory
```

I think this may be because the` futex.manifest` file I created in `test/runtime` is incorrect.
When I run `make -C test/runtime futex` from the top-level folder of the repo, my test compiles and I do `./output/test/runtime/bin/futex` to run the test. Do you know how to fix the issue with `make run TARGET=futex`?

Reply:
It seems your machine doesn't have KVM properly set up. You can try loading the KVM kernel module manually (with `sudo modprobe kvm-intel`) and see if that fixes the problem. Or alternatively run the futex test on Nanos without KVM acceleration: the command is `make run-noaccel TARGET=futex`

Reply:
Okay, I will try that. Thank you!